eidangservices/federator/share/wfcatalog.wadl

```diff
@@ -54,62 +54,326 @@
+            <param name="timing_correction_le" style="query" type="xsd:float"/>
 
             <!-- Miniseed Header Flags -->
             <!-- Data Quality Flags -->
             <param name="amplifier_saturation" style="query" type="xsd:float"/>
+            <param name="amplifier_saturation_eq" style="query" type="xsd:float"/>
+            <param name="amplifier_saturation_ne" style="query" type="xsd:float"/>
+            <param name="amplifier_saturation_gt" style="query" type="xsd:float"/>
+            <param name="amplifier_saturation_ge" style="query" type="xsd:float"/>
+            <param name="amplifier_saturation_lt" style="query" type="xsd:float"/>
+            <param name="amplifier_saturation_le" style="query" type="xsd:float"/>
             <param name="digitizer_clipping" style="query" type="xsd:float"/>
+            <param name="digitizer_clipping_eq" style="query" type="xsd:float"/>
+            <param name="digitizer_clipping_ne" style="query" type="xsd:float"/>
+            <param name="digitizer_clipping_gt" style="query" type="xsd:float"/>
+            <param name="digitizer_clipping_ge" style="query" type="xsd:float"/>
+            <param name="digitizer_clipping_lt" style="query" type="xsd:float"/>
+            <param name="digitizer_clipping_le" style="query" type="xsd:float"/>
             <param name="spikes" style="query" type="xsd:float"/>
+            <param name="spikes_eq" style="query" type="xsd:float"/>
+            <param name="spikes_ne" style="query" type="xsd:float"/>
+            <param name="spikes_gt" style="query" type="xsd:float"/>
+            <param name="spikes_ge" style="query" type="xsd:float"/>
+            <param name="spikes_lt" style="query" type="xsd:float"/>
+            <param name="spikes_le" style="query" type="xsd:float"/>
             <param name="glitches" style="query" type="xsd:float"/>
+            <param name="glitches_eq" style="query" type="xsd:float"/>
+            <param name="glitches_ne" style="query" type="xsd:float"/>
+            <param name="glitches_gt" style="query" type="xsd:float"/>
+            <param name="glitches_ge" style="query" type="xsd:float"/>
+            <param name="glitches_lt" style="query" type="xsd:float"/>
+            <param name="glitches_le" style="query" type="xsd:float"/>
             <param name="missing_padded_data" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_eq" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_ne" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_gt" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_ge" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_lt" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_le" style="query" type="xsd:float"/>
             <param name="telemtry_sync_error" style="query" type="xsd:float"/>
```

Comment:
Could you please fix the typo here, too (see also PR https://github.com/EIDA/wfcatalog/pull/15)? Thx.

Reply:
Done, thanks!

Reply:
Thanks guys.